src/net/codec.rs

```diff
@@ -902,6 +916,7 @@ impl StacksMessageType {
             StacksMessageType::Pong(ref _m) => StacksMessageID::Pong,
             StacksMessageType::NatPunchRequest(ref _m) => StacksMessageID::NatPunchRequest,
             StacksMessageType::NatPunchReply(ref _m) => StacksMessageID::NatPunchReply,
+            StacksMessageType::Attachment(ref _m) => StacksMessageID::Attachment,
```

Comment:
Attachments are not propagated through the p2p network, so there does not need to be an `Attachment` variant of `StacksMessageType`.  Please remove it.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/52b70da1a59b770c39484522f20698f478b90d1a